README.md

```diff
@@ -1351,6 +1345,10 @@ drive report-issue -title "Can't open my file" -body "Drive trips out every time
 cat bugReport.txt | drive issue -piped -title "push: dump on pushing from this directory"
 ```
 
+### Automating
```

Comment:
So, if I remove the duplicate link to the wiki, but keep a section about how to install the scripts and another one about how to use the scripts, like this:
```
## Installing
...
### Automation Scripts

Installing scripts which offer automated drive commands including syncing is described in [platform_packages.md](https://github.com/odeke-em/drive/blob/master/platform_packages.md)).

## Usage
...
### Automating

Using the automation scripts is described in [[drive-google wiki](https://gitlab.com/jean-christophe-manciot/Drive/wikis/home).
...
```